<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyPorta</class>
 <widget class="QDialog" name="modifyPorta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="portaLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Porta:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipoPortaLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo da porta:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nEntradasLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NÂ° de entradas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="id1Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID da entrada 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="id2Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID da entrada 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="id3Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID da entrada 3:</string>
   </property>
  </widget>
  <widget class="QLabel" name="id4Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID da entrada 4:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tipoPortaComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NOT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AND</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NAND</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NOR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XOR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NXOR</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="nEntradasSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="id1SpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="id2SpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="id3SpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="id4SpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="portaNumberLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>modifyPorta</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>modifyPorta</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
